content/knowledge-base/guides/composition-functions.md

```diff
@@ -815,11 +812,12 @@ ENTRYPOINT ["/venv/bin/python3", "main.py"]
 ```
 {{< /expand >}}
 
-Create and push the Function just like you would any Docker image:
+Create and push the Function just like you would any Docker image.
+
+Build the function.
 
 ```shell
```

Comment:
@plumbis should we have a copy first line only directive here?

Reply:
also, when `{copy-lines="1"}` is used, is it possible to render the clipboard icon vertically aligned with the top line, instead of the vertical center of the code block, giving a visual hint that it will copy just that top line?

Reply:
good catch Dan. Added!